vmm/src/lib.rs

```diff
@@ -1288,11 +1293,19 @@ impl Vmm {
                         EpollDispatch::DeviceHandler(device_idx, device_token) => {
                             METRICS.vmm.device_events.inc();
                             match self.epoll_context.get_device_handler(device_idx) {
-                                Ok(handler) => handler.handle_event(
-                                    device_token,
-                                    events[i].events,
-                                    EpollHandlerPayload::Empty,
-                                ),
+                                Ok(handler) => {
+                                    let r = handler.handle_event(
+                                        device_token,
+                                        events[i].events,
+                                        EpollHandlerPayload::Empty,
+                                    );
+                                    match r {
```

Comment:
`match` directly on the result of `handle_event` instead of moving it around through a variable